Fix bytes value from json

ebakoba opened the pull request:
Resolves the issue raised in issue #155

stephenh commented:
Thanks for the PR!

Could you change `integration/simple/simple.proto`'s `SimpleWithWrappers` to includes a `BytesValue`, rerun `update-bins.sh`, then rerun `./codegen.sh simple`, to reproduce and show the generated code works now?

Also adding a unit test in `simple-json.test.ts` that shows decoding bytes from JSON would be even better.

ebakoba commented:
@stephenh 
After the changes newly generated code adds the ```new``` keyword before the ```Uint8Array```, which is the correct behavior.
![image](https://user-images.githubusercontent.com/14753019/107120286-7064c480-6895-11eb-9f69-dfe4ff9aeddc.png)

I also updated the ```SimpleWithWrapper``` in ```integration/simple/simple.proto``` to include ```google.protobuf.BytesValue``` and updated tests to reflect that change.
